cypress/integration/brewery/ScenarioValidation.spec.js

```diff
@@ -4,38 +4,43 @@
 import utils from '../../commons/TestUtils';
 import { DATASET, RUN_TEMPLATE } from '../../commons/constants/brewery/TestConstants';
 import { Scenarios, ScenarioManager, ScenarioParameters, Login } from '../../commons/actions';
-
-Cypress.Keyboard.defaults({
-  keystrokeDelay: 0,
-});
+import { ALL_ROOT_SCENARIOS } from '../../fixtures/stubbing/ScenarioListOrder/scenarios';
+import { stub } from '../../commons/services/stubbing';
+import { setup } from '../../commons/utils/setup';
 
 const SCENARIO_DATASET = DATASET.BREWERY_ADT;
 const SCENARIO_RUN_TEMPLATE = RUN_TEMPLATE.BASIC_TYPES;
 
 describe('Scenario validation', () => {
   before(() => {
+    setup.initCypressAndStubbing();
     Login.login();
   });
 
   beforeEach(() => {
+    stub.setScenarios(ALL_ROOT_SCENARIOS);
+    stub.setFakeWorkspaceId('W-stbbdbrwry');
     Login.relogin();
   });
 
   const scenarioNamesToDelete = [];
+
   after(() => {
-    ScenarioManager.deleteScenarioList(scenarioNamesToDelete);
+    if (!stub.isEnabledFor('CREATE_AND_DELETE_SCENARIO')) {
+      ScenarioManager.deleteScenarioList(scenarioNamesToDelete);
+    }
   });
 
   it('can validate & reject scenarios', () => {
     let scenarioId;
     const prefix = 'Test Cypress - Scenario validation - ';
     const randomString = utils.randomStr(7);
     const scenarioName = prefix + randomString;
-    scenarioNamesToDelete.push(scenarioName);
+    if (!stub.isEnabledFor('CREATE_AND_DELETE_SCENARIO')) scenarioNamesToDelete.push(scenarioName);
```

Comment:
We should never have to call `stub.isEnabledFor` i na test file : we know exactly whether or not a stubbing type is enabled.
By the way, a call to `stub.start({   [...]  })` is missing in your test